src/ClockCountDown.elm

```diff
@@ -0,0 +1,102 @@
+module ClockCountDown exposing (Model, Msg(..), init, main, subscriptions, update, view)
+
+import Browser
+import Html exposing (..)
+import Task
+import Time
+
+
+main =
+    Browser.element
+        { init = init
+        , view = view
+        , update = update
+        , subscriptions = subscriptions
+        }
+
+
+type alias Model =
+    { zone : Time.Zone
+    , time : Time.Posix
+    }
+
+
+type Msg
+    = Tick Time.Posix
+    | AdjustTimeZone Time.Zone
+    | CurrTime Time.Posix
+
+
+init : () -> ( Model, Cmd Msg )
+init _ =
```

Comment:
Instead of ignoring an argument inside `init`, use `always`:
```elm
init : (Model, Cmd Msg)
init = ...
```
and in `main`:
```elm
    Browser.element
        { init = always init
        ...
```